pkg/cmd/server/origin/controller/config.go

```diff
@@ -27,7 +27,7 @@ func getOpenShiftClientEnvVars(options configapi.MasterConfig) ([]kapi.EnvVar, e
 	if err != nil {
 		return nil, err
 	}
-	return clientcmd.EnvVars(
+	return clientconfig.EnvVars(
```

Comment:
This appears to be the only caller.  Move it locally.

Reply:
Actually the entire `getOpenShiftClientEnvVars` needs refactoring.  Queue up an issue to tidy it up.

Reply:
Opened issue to track this: https://github.com/openshift/origin/issues/17397